HeliPort/Appearance/StatusMenu.swift

```diff
@@ -249,10 +252,8 @@ class StatusMenu: NSMenu, NSMenuDelegate {
         print(sender.title)
         switch sender.title {
         case NSLocalizedString("Turn Wi-Fi On", comment: ""):
-            isNetworkEnabled = true
             power_on()
         case NSLocalizedString("Turn Wi-Fi Off", comment: ""):
-            isNetworkEnabled = false
```

Comment:
I wold advise if you do end up removing this to also remove this  condition under `func updateStatus() ` since if the Wi-Fi state is off and you're trying to turn on the wifi, the condition will always fail since nothing sets the `isNetworkEnabled = true`.
 https://github.com/zxystd/HeliPort/blob/91b759e67e1beb19b9633d166f6f1c8a72d93031/HeliPort/Appearance/StatusMenu.swift#L314-L316

Reply:
thanks